ANSWERS.md

```diff
@@ -1,4 +1,17 @@
 - [ ] Why would you use class component over function components (removing hooks from the question)?
+
```

Comment:
If we're forgetting that hooks exist (wouldn't that be sad) you'd want to use class components when you need to hold state or use lifecycle methods because you can't do either with functional components without hooks.